### ISSUE TITLE
Undefined variable: priority class.redux_api.php with Redux::removeField

### ISSUE THREAD

**Issue by dimitrilongo** [reduxframework/redux-framework#2473]
hi,

just play with API, and found that issue :
```Notice: Undefined variable: priority in /SVN/redux-test/wp-content/themes/redux-theme/admin/redux-framework/inc/class.redux_api.php on line 372```
it works, but i have that notice.

on a fresh install of wp (made with redux builder).
my section :
```php 
Redux::setSection( $opt_name, array(
                    'title'  => __( 'Config', 'domain' ),
                    'desc'   => __( '', 'domain' ),
                    'icon'   => ' el el-adjust-alt',
                    'fields' => array(
							array(
								'id'        => 'el-args-pos-bar',
								'type'      => 'slider',
								'title'     => __('Custom position in admin bar', 'domain'),
								'subtitle'  => __('', 'domain'),
								'desc'      => __('', 'domain'),
								"default"   => 50,
								"min"       => 0,
								"step"      => 5,
								"max"       => 150,
								'display_value' => 'text',
							), 
							array(
								'id'		=>'el-section-remove-help',
								'type' 		=> 'switch', 
								'title' 	=> __('Remove Help Section', 'domain'),
								'subtitle'	=> __('', 'domain'),
								"default" 	=> '1',
								'on' 		=> __('Show', 'domain' ),
								'off' 		=> __('Hide', 'domain' ),
								),	
							),
                 ) );
```
and finally to remove 
```Redux::removeField($opt_name, 'el-section-remove-help');```
In the section if field ```'id'		=>'el-section-remove-help',``` is the first one (no fields before it) I have no php notice.
Don't know if it helps

Regards

dev_mode off
wp 4,2,2
redux embed 3,5,5,4
wp_debug true
mamp

**Comment by dovy** [comment]
Two issues here. One, I need to add a proper check. Two, you didn't declare the section ID.  ;)

With the Redux API sections need IDs. But i'll fix the error notice in the API, because we do auto-generate section IDs if you don't supply it.  ;)

**Comment by dovy** [comment]
Fixed in 3.5.5.5

**Comment by dimitrilongo** [comment]
thanks ;)
